app/src/main/java/org/listenbrainz/android/util/Constants.kt

```diff
@@ -10,11 +10,7 @@ object Constants {
     const val FEEDBACK_EMAIL = "support@metabrainz.org"
     const val FEEDBACK_SUBJECT = "[LBAndroid] Feedback"
     const val SPOTIFY_REDIRECT_URI = "org.listenbrainz.android://callback"
-    const val MUSICBRAINZ_AUTH_BASE_URL = "https://musicbrainz.org/oauth2/"
     const val LISTENBRAINZ_API_BASE_URL = "https://api.listenbrainz.org/1/"
```

Comment:
I think we should remove the `1/` now from here?

Reply:
the website does this too afaik. Plus the `1/` is just repeated unnecesaarily

Reply:
It's a way of version tracking the backend. Let me confirm with the team what the status is. If we have multiple versions or have resolved to a single version now.

Reply:
What should we do then?

Reply:
lucifer mentioned that `/1` is the go-to. Let's stick to it for now.

Reply:
yippe